src/mpyl/steps/deploy/k8s/resources/traefik.py

```diff
@@ -27,34 +28,45 @@ def full_name(self) -> str:
 
 
 class V1AlphaIngressRoute(CustomResourceDefinition):
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         metadata: V1ObjectMeta,
         host: HostWrapper,
         target: Target,
         namespace: str,
+        cluster_env: str,
         pr_number: Optional[int],
+        middlewares_override: list[str],
+        entrypoints_override: list[str],
         https: bool = True,
     ):
-        def _interpolate_names(host: str, name: str) -> str:
+        def _interpolate_names(host: str, name: str, cluster_env: str) -> str:
             host = host.replace("{SERVICE-NAME}", name)
             host = host.replace("{namespace}", namespace)
+            host = host.replace("{CLUSTER-ENV}", cluster_env)
             host = replace_pr_number(host, pr_number)
             return host
 
+        combined_middlewares = (
+            [
+                {"name": "traefik-https-redirect@kubernetescrd"} if not https else None,
```

Comment:
I know this was already hardcoded, but should it be?

Reply:
no it probably shouldn't be. Do you want me to change it here?

Reply:
Normally I would say you could create a new ticket, but this kinda fits within the scope of this pr right?

Reply:
done in [e4b7a9b](https://github.com/Vandebron/mpyl/pull/335/commits/e4b7a9be824e2c3b9d273d14fc5bf0a447a2ed93)